### ISSUE TITLE
Maven Error

### ISSUE THREAD

**Issue by loren138** [iluwatar/java-design-patterns#534]
I was trying to list a dependency tree for your project using ``mvn dependency:tree`` but I keep getting the following error:

> Failed to execute goal on project naked-objects-fixture: Could not resolve dependencies for project com.iluwatar:naked-objects-fixture:jar:1.14.0: Failure to find com.iluwatar:naked-objects-dom:jar:1.14.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

I removed the ``naked-objects`` module from the main pom.xml file and then it worked for the rest of the repository so I think this is an issue in one of the pom.xml files in the ``naked-objects`` folder, but if it is an issue in my usage of the repository, please let me know.  (I'm fairly new to java and maven so I was not sure how to proceed any further with debugging or fixing.)

**Comment by iluwatar** [comment]
Seems to work for me (see the attached log file). Can you update to the latest source codes, run `git clean -xfd` and `mvn clean` and then `mvn dependency:tree`?

**Comment by loren138** [comment]
That is strange.  I'm running Ubuntu 16.04

`you@downloadgit:~/java-design-patterns$ mvn --version
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 1.8.0_111, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-openjdk-amd64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-57-generic", arch: "amd64", family: "unix"
you@downloadgit:~/java-design-patterns$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
you@downloadgit:~/java-design-patterns$ git clean -xfd
you@downloadgit:~/java-design-patterns$ mvn clean
- Output omitted
you@downloadgit:~/java-design-patterns$ mvn dependency:tree > out.txt`

Here's the result of the java run: http://klingmandesign.com/out.txt

The machine is a clean install for a project I'm working on so it shouldn't have any artifacts.

**Comment by loren138** [comment]
I've solved it.  I had not run `mvn install` and maven is unable to build a dependency tree with the intra-project dependencies unless the project has been installed first.

Once `mvn install` has been run for that particular version, the dependency tree for that version can be built successfully for the project as a whole.

(Thanks to SO for the tip http://stackoverflow.com/questions/7953888/how-to-get-maven-dependencies-printed-to-a-file-in-a-readable-format and to @iluwatar for confirming that it should work such that I did a little more googling to find a solution.)
